include/dlaf/eigensolver/band_to_tridiag/mc.h

```diff
@@ -341,7 +485,7 @@ class SweepWorker {
   }
 
   T& tau() noexcept {
-    return *data_(0);
+    return *data_();
   }
   const T& tau() const noexcept {
     return *data_(0);
```

Comment:
Also `data_()` as above, or is this intentionally left different?

Reply:
It is a reminder of debugging... Fixed.